_meta/kibana/5.x/visualization/1bdca740-7828-11e7-8c47-65b845b5cfb3.json

```diff
@@ -1,6 +1,6 @@
 { 
   "title": "Top Applications by Response Time [APM]",
```

Comment:
You should probably also rename the titles from `Application` to `Service` in all the visualizations and dashboards.